scripts/build/create_commands.py

```diff
@@ -18,61 +18,30 @@
 LOG.addHandler(log_handler)
 
 
-def copy_entry_points(input_data, build_dir):
-    """
-    Copy CodeChecker entry point sub-commands.
-    """
-    package_root = os.path.join(build_dir, 'CodeChecker')
-    package_bin = os.path.join(package_root, 'bin')
-    package_bin_cmd = os.path.join(package_bin, 'subcmd')
-    os.makedirs(package_bin_cmd, exist_ok=True)
-    target_cc = os.path.join(package_root, 'cc_bin')
-
-    available_commands = {}
-    for i in input_data:
-        module_name = i.split(':')[1]
-        input_dir = i.split(':')[0]
-
-        for input_file in glob.glob(os.path.join(input_dir, '*')):
-            file_name = os.path.basename(input_file)
-            if not file_name.endswith(".py"):
-                # Non-py files use the environment to appear as python
-                # files, they go into the folder in PATH as they are
-                # entry points.
-                if file_name.startswith("codechecker-"):
-                    command_name = file_name.replace("codechecker-", "")
-
-                    module_path = module_name + '/' + \
-                        command_name.replace('-', '_') + '.py'
-                    available_commands[command_name] = module_path
-
-                    skip_content = "# DO_NOT_INSTALL_TO_PATH"
-                    with open(input_file, 'r',
-                              encoding="utf-8", errors="ignore") as file:
-                        if file.readline().strip() == skip_content:
-                            # If the file is marked not to install, do not
-                            # install it. This happens with entry points
-                            # whom should not act as "lowercase" entries,
-                            # but the sub-command exists as an available
-                            # command.
-                            continue
-
-                # Only the CodeChecker top level script should be copied
-                # into the bin directory and every other subcommand should be
-                # called through that.
-                if file_name.startswith("codechecker-"):
-                    shutil.copy2(input_file, package_bin_cmd)
-                else:
-                    shutil.copy2(input_file, package_bin)
-
-            else:
-                # .py files are Python code that must run in a valid env.
-                shutil.copy2(input_file, target_cc)
-
-    commands_json = os.path.join(target_cc, 'commands.json')
+def collect_subcmd(cmd_dirs, build_dir):
+    subcmds = {}
```

Comment:
Previously we put sub commands under `build/CodeChecker/bin/subcmd/`. Now this is removed. My question is we used files under `bin` directory (e.g.: `web/client/bin/codechecker-store`) to create subcommands only?

Reply:
Yes, these files were used only for assembling the content of `commands.json`, and these scripts were never executed.